ssh.go

```diff
@@ -147,10 +156,18 @@ func defaultAuthMethods(user, identity string, agt agent.Agent) map[string]ssh.A
 		}
 	}
 
+	for _, i := range identityFiles {
+		if m, err := newSSHPublicKeyAuthMethod(i); err == nil {
+			methods[i] = m
+		} else {
+			log.Warningf("Failed to load identity file %s", i)
```

Comment:
Would this print out too many warnings when people only have one type of identity file?